spec/ndx-EXTRACT.extensions.yaml

```diff
@@ -3,232 +3,406 @@ groups:
   neurodata_type_inc: ImageSegmentation
   doc: EXTRACT configuration parameters
   attributes:
+  - name: version
+    dtype: text
+    doc: The version of EXTRACT.
+    required: false
   - name: preprocess
-    dtype: int32
-    doc: int32ean flag indicating data preprocessing before main EXTRACT function
+    dtype: bool
+    doc: Indicates whether or not data preprocessing was applied before main EXTRACT
+      function.
+    required: false
+  - name: pre_mask_on
+    dtype: bool
```

Comment:
In MATLAB this say's it is encoded as a double (float64), but I doubt it would ever be a value other than 0 or 1, so fine to keep as bool. Just leaving this as a note